src/main/java/com/hivemq/metrics/HiveMQMetrics.java

```diff
@@ -187,5 +187,13 @@ public class HiveMQMetrics {
     public static final HiveMQMetric<Counter> WILL_MESSAGE_PUBLISHED_COUNT_TOTAL =
             HiveMQMetric.valueOf("com.hivemq.messages.will.published.count.total", Counter.class);
 
+    /**
+     * represents a {@link Counter}, which measures the current count of shared subscriptions
+     *
+     * @since 2022.2
```

Comment:
is correct?

Reply:
We don't typically add new metics to the community edition. I'm not personally opposed to it, but I remember that it was once an explicit decision to have advanced metrics as an enterprise feature or something.

Reply:
removed along with one metric that was not used at all